deployments/types/rococo.json

```diff
@@ -0,0 +1,14 @@
+{
+	"Fee": "RococoBalance",
+	"Balance": "RococoBalance",
+	"BlockHash": "RococoBlockHash",
+	"BlockNumber": "RococoBlockNumber",
+	"BridgedBlockHash": "WococoBlockHash",
```

Comment:
I've started wondering how are we going to handle multiple bridges with **different** types here, but it's more of a Metadata/UI problem really.